internal/core/media.go

```diff
@@ -258,11 +259,13 @@ func (c *Community) UploadFile(ctx *yap.Context) {
 		ctx.JSON(500, err.Error())
 		return
 	}
+	// todo Use configuration form in the future
+	var maxFileSize int64 = 300 * 1024 * 1024
```

Comment:
Maybe we can set this limit to config to avoid hard code.

Reply:
> Maybe we can set this limit to config to avoid hard code.也许我们可以将此限制设置为 config 以避免硬代码。

Probably the best approach is to dynamically configure